src/amber/cli/templates/app/docker-compose.yml.ecr

```diff
@@ -15,36 +15,29 @@ services:
   app:
     build: .
     image: <%= @name %>
-    command: 'amber watch'
-    working_dir: /app/local
+    command: amber watch
     environment:
       DATABASE_URL: <%= @db_url %>
-      SMTP_URL: mail:1025
     ports:
-      - '3000:3000'
+      - 3000:3000
     links:
       - db
-      - mail
     volumes:
-      - '.:/app/local'
+    - .:/app
+    - nodes:/app/node_modules
+    - shards:/app/lib
```

Comment:
This is pretty cool feature I just found out.  We can add a volume to the node_modules and the shards directories.  This removes the need to run `npm install` and `shards install` locally.  It will use the `/lib` and `/node_modules` on the docker image.

now you can get up to speed even faster:
```bash
amber new blog
cd blog
docker-compose up -d
```